packages/client/src/Container.ts

```diff
@@ -1,12 +1,14 @@
 import 'reflect-metadata'
 import './utils/PatchTsyringe'
 import { DependencyContainer } from 'tsyringe'
-import { uuid } from './utils/uuid'
-import { counterId } from './utils/utils'
-import { Debug } from './utils/log'
-import { Context } from './utils/Context'
 import { ConfigInjectionToken, StrictStreamrClientConfig } from './Config'
 import { AuthenticationInjectionToken, createAuthentication } from './Authentication'
+import { counterId } from './utils/utils'
+import { uuid } from './utils/uuid'
+
+function generateClientId(): string {
+    return counterId(process.pid ? `${process.pid}` : `${uuid().slice(-4)}${uuid().slice(0, 4)}`)
```

Comment:
If uuid is just a random string we could just get first n chars from that? Or if it is not a random (but e.g. a time-based) uuid, should we change that to be a random id?

Reply:
Just changed to use `randomString` in 648e1fd356c24ec565b9d6e69f35418cb2f22bbd